R/PseudoBulk.R

```diff
@@ -1184,41 +1187,54 @@ FitRegularizedClassificationGlm <- function(seuratObj,
   if (is.null(split)){
     split <- mlr3::partition(task_metadata_classification)
   }
+
+  print('DEBUG!!!')
+  print(table(target_labeled_data[[metadataVariableForClassification]]))
+  print(split)
+  print('TRAINING:')
+  print(table(target_labeled_data[[metadataVariableForClassification]][split$train]))
+  print('TEST:')
+  print(table(target_labeled_data[[metadataVariableForClassification]][split$test]))
```

Comment:
looks like this catches it: 
control      unvax vaccineOne vaccineTwo 
         2          2          3          1
from what I understood from your traceback, the vaccineTwo value is now a problem. 

I think I can do a better job on the resampling than mlr3::partition().

Reply:
circling back, there are some issues with non-randomness when you do basic assurance on the resampling (like the number of observations in each class > 1 per split). 

I increased the number of synthetic samples for the tests to ensure that the number of samples is more realistic to fix this.